packages/wellknown-did-issuer/src/entities/DidConfigurationResourceEntity.ts

```diff
@@ -0,0 +1,91 @@
+import { DomainLinkageCredential, IDidConfigurationResource, ISignedDomainLinkageCredential } from '@sphereon/wellknown-dids-client'
+import { VerifiableCredential } from '@veramo/core'
+import { Credential, Identifier, Claim } from '@veramo/data-store'
+import { asArray, computeEntryHash, extractIssuer } from '@veramo/utils'
+import { Entity, Column, PrimaryColumn, CreateDateColumn, UpdateDateColumn, ManyToMany, JoinTable, BaseEntity } from 'typeorm'
+
+@Entity('DidConfigurationResource')
+export class DidConfigurationResourceEntity extends BaseEntity {
+  @PrimaryColumn({ nullable: false })
+  origin!: string
+
+  @Column({ nullable: false })
+  context!: string
+
+  // TODO cascade of delete should be true when the VC is only attached to one DID configuration resource
+  @ManyToMany(() => Credential, (credential: Credential) => credential.hash, { nullable: false, cascade: true, onDelete: 'NO ACTION' })
+  @JoinTable({ name: 'DidConfigurationResourceCredentials' })
+  linkedDids!: Array<Credential>
+
+  @CreateDateColumn({ name: 'created_at', nullable: false })
+  createdAt!: Date
```

Comment:
Interesting. Did you test this with both SQLite and Postgres. I couldn't make the date time work on both using timestamps I believe, so I had to resort to storing them as strings and using a custom converter in the entity

Reply:
Like discussed, seems to be working on both with the use of Dates